v2/components/schemas/Media.yaml

```
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: GPL-3.0
+# SPDX-FileCopyrightText: Canonical Ltd
+
+type: object
+description: A media asset for the snap, such as an icon or screenshot.
+properties:
+  height:
+    type: integer
+    description: |-
+      The height of the asset in pixels.
+  type:
+    type: string
+    enum:
+      - icon
+      - screenshot
+  url:
+    type: string
+    format: uri
+    description: |-
+      The URL for the asset.
```

I'm not sure but this reads better to my non-native ears:

```suggestion
      The URL of the asset.
```